bugbug/similarity.py

```diff
@@ -78,22 +79,29 @@ def __init__(self, cleanup_urls=True, nltk_tokenizer=False):
     def get_text(self, bug):
         return "{} {}".format(bug["summary"], bug["comments"][0]["text"])
 
-    def text_preprocess(self, text, join=False):
+    def text_preprocess(self, text, spacy_lemmatization=False, join=False):
+        nlp = spacy.load("en_core_web_sm")
```

Comment:
It'd be better to do the loading only once (at the top of the file in a global variable so you can use it everywhere), and not for every call to text_preprocess